cunumeric/module.py

```diff
@@ -7545,6 +7545,144 @@ def var(
     )
 
 
+@add_boilerplate("m", "y", "fweights", "aweights")
+def cov(
```

Comment:
Add this function to the appropriate .rst file under the docs folder, otherwise the documentation won't build.